src/Auth/Login.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { RouteComponentProps, withRouter } from "react-router-dom";
+
+import Input from "src/components/Input";
+import * as routes from "src/common/constants/pageRoutes";
+import Button from "src/components/Button";
+import Link from "src/components/Link";
+import { SignInContainer } from "./Auth.styles";
+import { IParams } from "src/common/interface";
+
+const Login: React.FunctionComponent<RouteComponentProps<IParams>> = ({
+  history,
+}) => {
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+
+  const handleInputChange = ({
+    target: { name, value },
+  }: React.ChangeEvent<HTMLInputElement>) => {
+    switch (name) {
+      case "email":
+        setEmail(value);
+        break;
+      default:
+        setPassword(value);
+    }
+  };
+
+  const handleSubmit = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    history.push(routes.HOME);
+  };
```

Comment:
This function can be pulled out of the component, to avoid creating a new object on each render

Reply:
right now it does nothing but it will use components's `email` and `password` state for login